.jshintrc

```diff
@@ -30,5 +30,5 @@
   "white": false,
   "eqnull": true,
   "esnext": true,
-  "unused": "vars"
+  "unused": true
```

Comment:
Yep, was just matching ember-cli standards, but I think it's totally acceptable to do this.